addons/interaction/CfgVehicles.hpp

```diff
@@ -251,6 +251,15 @@ class CfgVehicles {
                 statement = QUOTE([ARR_3(_player,_target,1)] call DFUNC(tapShoulder));
                 exceptions[] = {"isNotSwimming"};
             };
+            class ACE_OpenBackpack {
+                displayName = CSTRING(OpenBackpack);
+                position = QUOTE(call DFUNC(getBackpackPos));
+                distance = 1.50;
+                condition = QUOTE(!isNull backpackContainer _target);
```

Comment:
from ace_medical_status_fnc_addInventoryActions

probably also want to check these, handling UAV backpacks or locked
```
    {!lockedInventory _backpackContainer} &&
    {maxLoad _backpackContainer > 0} &&
    {getNumber (_backpackConfig >> "disableInventory") != 1} &&
    {!(_target isKindOf QEGVAR(dragging,clone))} &&
```

Reply:
I forgot that you could even lock inventories, makes sense